features/nth-child.yml.dist

```diff
@@ -40,3 +79,16 @@ compat_features:
   #   safari: "3.1"
   #   safari_ios: "2"
   - css.selectors.nth-last-child
+
+  # baseline: false
+  # support:
+  #   chrome: "57"
+  #   chrome_android: "57"
+  #   edge: "79"
+  #   firefox: "52"
+  #   firefox_android: "52"
+  - css.selectors.first-child.no_parent_required
+  - css.selectors.last-child.no_parent_required
+  - css.selectors.nth-child.no_parent_required
+  - css.selectors.nth-last-child.no_parent_required
+  - css.selectors.only-child.no_parent_required
```

Comment:
I'm unclear as to what these subfeatures cover. Can you help me understand?
Would they make sense as a separate feature?

Reply:
Selectors level 3 had text referring to the children's parent, but selectors level 4 adds the note-

```
As there was no reason to exclude them from matching elements without parents, or with non-element parents, they have been rephrased to refer to an element’s relative index amongst its siblings.
```

For instance, the document root matches `:first-child` despite having no parent. 

This doesn't seem like a feature to me. It also looks like Safari data might be incorrect, as `document.documentElement.matches(":first-child")` returns true in Safari.

Reply:
Thanks! Keeping them here, ignored, seems right then.